css/style.css

```diff
@@ -0,0 +1,193 @@
+* {
+    -webkit-box-sizing:border-box;
+    box-sizing:border-box;
+    border: black solid 1px;
+}
+
+html {
+    font-size: 62.5%;
+    margin: 0;
+}
+
+body {
+    margin: auto;
+    font-family: 'Inconsolata', monospace;
+    font-size: 14px;
+}
+
+a {
+    color: #000;
+    padding: 4px
+}
+
+a:link {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+header {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
```

Comment:
Your header doesn't need to be flex, it does nothing. You can remove all of the flex properties and prefixes.